src/libs/OptionsListUtils.js

```diff
@@ -64,6 +64,7 @@ function getSearchText(report, personalDetailList) {
     });
 
     if (report) {
+        searchTerms.push(report.reportName);
```

Comment:
Which issue does this fix?

Reply:
This fixes the linked issue, involving the ability to use commas when searching for group chats with commas in their names. Below this line we were splitting the group chat's name by commas and adding it to the search terms, but we were leaving out the actual name of the group chat with commas.

Reply:
Hmm sorry I don't really get the solution. Does it only work when you type in the exact beginning of the report name? e.g. if you switch up the order does it no longer work?

Reply:
In your video you type `Jasper Taylor, nonadmin@expensify.com` what happens when you do `nonadmin@expensify.com, Jasper Taylor`?

Reply:
Hey Marc, sorry for missing this!

It should still work if you switch up the order of the names.

Whenever a user types in the search input, we use [this regex](https://github.com/Expensify/Expensify.cash/blob/16a570fab5f9ba95dd96c6cc36c0b5e166416f03/src/libs/OptionsListUtils.js#L145-L152) to check that input against search terms for each option. Previously, if you searched `Jasper Taylor, nonadmin@expensify.com`, nothing would appear because the search terms for that chat would be this. They're doubled because [these two lines](https://github.com/Expensify/Expensify.cash/blob/16a570fab5f9ba95dd96c6cc36c0b5e166416f03/src/libs/OptionsListUtils.js#L67-L68) are equivalent for this case.
```
['Jasper', 'nonadmin@expensify.com', 'Jasper', 'nonadmin@expensify.com'];
```

This change just makes it so we add the name with commas as a search term:
```
['Jasper Taylor, nonadmin@expensify.com', 'Jasper', 'nonadmin@expensify.com', 'Jasper', 'nonadmin@expensify.com'];
```